plugins/easyimage/styles/easyimage.css

```diff
@@ -43,6 +43,13 @@ The outline is not a part of the element's dimensions, we have to use a border a
 	max-width: 25%;
 }
 
+/*
+Overwrite 25% in classic editor to prevent existing this bug https://bugs.chromium.org/p/chromium/issues/detail?id=803045
```

Comment:
I would go with something like:

> Overwrite `max-width: 25%` with `max-width: 25vw` to prevent native Chrome bug - https://bugs.chromium.org/p/chromium/issues/detail?id=803045 (#1550).